handlers/user.go

```diff
@@ -0,0 +1,139 @@
+package handlers
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"strconv"
+
+	"github.com/somos831/somos-backend/models"
+	"github.com/somos831/somos-backend/responses"
+)
+
+const (
+	UserNotFoundErr = "user not found"
```

Comment:
We could have our error messages be actual errors instead
```go
const (
  UserNotFoundErr = errors.New("user not found")
)
```
This way when checking the type of an error, we can do
```go
if errors.Is(err, UserNotFoundErr) {
  // Handle user not found 
}
```
instead of 
```go
if err != nil && err.Error() == UserNotFoundErr {
  // Handle user not found
}
```